### PR TITLE
Add e2e test for automatic sidecar injection

### PR DESCRIPTION
**Pull request by yusuoh**

### CODE REVIEW

File: tests/e2e/framework/kubernetes.go [hunk 1]
```diff
@@ -493,15 +500,14 @@ func (k *KubeInfo) deployAddons() error {
...
-			istioYaml = authInstallFile
-		} else {
-			istioYaml = nonAuthInstallFile
-		}
+		istioYaml = getClusterWideInstallFile()
 	} else {
 		if *authEnable {
 			istioYaml = authInstallFileNamespace
 		}
+		if *useAutomaticInjection {
```
Comment (ayj) [inline tests/e2e/framework/kubernetes.go]: +1

**Comment by chxchx** [comment]
ping

**Pull request closed without merging**